dev-requirements.txt

```diff
@@ -0,0 +1,4 @@
+flask==1.0.2
+mock
+pytest-ckan
+pytest-cov
```

Comment:
I guess we need ckanext-scheming as a requirement here

Reply:
Added to the requirements file